Samples/excel-add-in-shape-dashboard/src/taskpane/taskpane.ts

```diff
@@ -0,0 +1,431 @@
+    shapes.load("items");
+    await context.sync();
+
+    let shapeGroup = shapes.items[0].group.shapes;
+    shapeGroup.load("items");
+    await context.sync();
+
+    let text2 = context.workbook.worksheets.getItem("Sample").getRange("E152:E152");
+    text2.load("values");
+    await context.sync();
+
+    let region = shapeGroup.items[0];
+    region.textFrame.textRange.text = Math.ceil(parseFloat(text2.values.toString())).toString() + "\nRegions";
+    region.fill.foregroundColor = "#339933";
+
+    let text3 = context.workbook.worksheets.getItem("Sample").getRange("E157:E157");
+    text3.load("values");
+    await context.sync();
+    let state = shapeGroup.items[1];
+    state.fill.foregroundColor = "#339933";
+    state.textFrame.textRange.text = text3.values.toString() + "\nStates";
+
+    let text4 = context.workbook.worksheets.getItem("Sample").getRange("E153:E153");
+    text4.load("values");
+    await context.sync();
+    let category = shapeGroup.items[3];
+    category.fill.foregroundColor = "#003366";
+    category.textFrame.textRange.text = text4.values.toString() + "\nCategories";
+
+    let text5 = context.workbook.worksheets.getItem("Sample").getRange("E154:E154");
+    text5.load("values");
+    await context.sync();
+    let subCategory = shapeGroup.items[4];
+    subCategory.fill.foregroundColor = "#003366";
+    subCategory.textFrame.textRange.text = text5.values.toString() + "\nSub Categories";
+
+    let text6 = context.workbook.worksheets.getItem("Sample").getRange("E155:E155");
+    text6.load("values");
+    await context.sync();
+    let maxSale = shapeGroup.items[6];
+    maxSale.fill.foregroundColor = "#FF6600";
+    maxSale.textFrame.textRange.text = text6.values.toString() + "\nMax Sale";
+
+    let text7 = context.workbook.worksheets.getItem("Sample").getRange("E156:E156");
+    text7.load("values");
+    await context.sync();
+    let sumSale = shapeGroup.items[7];
+    sumSale.fill.foregroundColor = "#FF6600";
+    sumSale.textFrame.textRange.text = text7.values.toString() + "\nSum Sale";
+
+    await context.sync();
+    showStatus("Add information to dashboard - success!", false);
+  });
+}
+
+async function changeFontFormat() {
+  await Excel.run(async (context) => {
+    showStatus("Running...", false);
+    let shapes = context.workbook.worksheets.getItem("Sample").shapes;
+    shapes.load("items");
+    await context.sync();
+
+    let shapeGroup = shapes.items[0].group.shapes;
+    shapes.items[0].onActivated.add(() => tryCatch(onActivate));
+    shapes.items[0].onDeactivated.add(() => tryCatch(onDeactivate));
+    shapeGroup.load("items");
+    await context.sync();
+
+    for (let i = 0; i < shapeGroup.items.length; i++) {
+      let shp = shapeGroup.items[i];
+      shp.textFrame.textRange.font.name = "Consolas";
+
+      shp.textFrame.verticalAlignment = Excel.ShapeTextVerticalAlignment.middle;
+      shp.textFrame.horizontalAlignment = Excel.HorizontalAlignment.center;
+    }
```

Comment:
Style nit: we typically favor `foreach` for code like this. Even if you don't change the loop, please change `shp` to `shape`.